src/js/popup.js

```diff
@@ -117,6 +121,15 @@ async function showRelayPanel(tipPanelToShow) {
     emailsBlockedValEl.textContent = emailsBlockedVal;
     emailsForwardedValEl.textContent = emailsForwardedVal;
 
+    //Show premium panel state
+    if (premium && premiumEnabledString === "True"){
```

Comment:
This if statement un-hides premium UI blocks if the user is premium and if premium features are available.